.circleci/config.yml.jinja

```diff
@@ -506,13 +506,13 @@ jobs:
                         echo "Should Publish Docker: $CODA_WAS_PUBLISHED"
                         set -x
                         if [[ "$CODA_WAS_PUBLISHED" = true  ]]; then
-
                               echo "$DOCKER_PASSWORD" | docker login --username $DOCKER_USERNAME --password-stdin
+                              {%- for docker_image in docker_images %}
                               scripts/release-docker.sh \
-                              -s {{docker_image}} \
-                              -v $CODA_GIT_TAG-$CODA_GIT_BRANCH-$CODA_GIT_HASH \
-                              --extra-args "--build-arg coda_version=$CODA_DEB_VERSION --build-arg deb_repo=$CODA_DEB_REPO"
-
+                                -s {{docker_image}} \
```

Comment:
Can you also define this for the puppeteered image in buildkite/scripts/docker-artifact.sh so that we can have buildkite + circleCI in the same PR? https://github.com/CodaProtocol/coda/blob/develop/buildkite/scripts/docker-artifact.sh#L8